singlestoredb/management/files.py

```diff
@@ -0,0 +1,465 @@
+#!/usr/bin/env python
+"""SingleStoreDB Files Management."""
```

Comment:
You might want to change this from SingleStoreDB to SingleStore Cloud. SingleStoreDB is used to refer to the database engine.